proto/api/v2/report_service.proto

```diff
@@ -98,7 +98,7 @@ service ReportService {
     // GetReportHistory returns the full history for a report configuration with the specified ID.
     rpc GetReportHistory(GetReportHistoryRequest) returns (ReportHistoryResponse) {
         option (google.api.http) = {
-            get: "/v2/reports/history/{report_config_id}"
+            get: "/v2/reports/configurations/{id}/history"
```

Comment:
Minor : Should we move this to `proto/api/v2/report_configuration_service.proto` since it is based on configuration now? 

I am perfectly fine keeping it here too.

Reply:
Good point. All report APIs should be provided under one service. There are two services catering the reporting feature, and there is not need to have them separate especially when they are tightly coupled. Once we align all the APIs with the guidelines, we should consolidate the two, hopefully soon.